CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+### Unreleased
+- Update requirements to include hashes
```

Comment:
Needs a better message - should write from the point of view of how it affects a consumer of the service. Just saying you're adding hashes isn't telling my what that means :) Something more like:
```
- Ensure integrity and version of library dependencies
```